resources/templates/ha-vpn-interop-guide-template/index.md

```diff
@@ -0,0 +1,836 @@
+the base Layer 3 network configuration of <vendor-name>. Note the following:
+
+-  At least one internal-facing network interface is required in order to
+connect to your on-premises network, and one external-facing interface is
+required in order to connect to GCP.
+
+```
+<insert configuration code snippet here>
+```
+
+### Creating the base VPN gateway configuration
+
+Follow the procedures in this section to create the base VPN configuration.
+
+<This section contains outlines of subsections for different aspects of
+configuring IPsec and IKE on the vendor side. Fill in the sections that are
+relevant to the current configuration, and remove any sections that don't
+apply.>
+
+### GCP-compatible settings for IPSec and IKE
+
+ [GCP support ciphers](https://cloud.google.com/vpn/docs/how-to/configuring-peer-gateway#configuring_ike).
+
+#### Configure the IKE proposal and policy
+
+<Insert the instructions for creating the IKE proposal and policy here. Below
+are some examples of IKE algorithms to specify as part of the instructions.>
+
+-  **Encryption algorithm**—< list required algorithms here>
+-  **Integrity algorithm**—< list required algorithms here>
+-  **Diffie-Hellman group—**< list required group here>
+
+```
+<insert configuration code snippet here>
+```
+
+#### Configure the IKEv2 keyring
+
+< insert the instructions for creating the IKEv2 keyring here.>
+
+```
+<insert configuration code snippet here>
+```
+
+#### Configure the IKEv2 profile
+
+< insert the instructions for creating the IKEv2 profile here.>
+
+```
+<insert configuration code snippet here>
+```
+
+#### Configure the IPsec security association (SA)
+
+< insert the instructions for creating the IPsec SA here. Below is an example of
+parameters to set.>
+
+-  **IPsec SA replay window-size**—Set this to 1024, which is the
+recommended value for <vendor-name><product-name>.
+
+```
+<insert configuration code snippet here>
+```
+
+#### Configure the IPsec transform set
+
+< insert the instructions for creating the IPsec transform set here.>
+
+```
+<insert configuration code snippet here>
+```
+
+#### Configure the IPsec static virtual tunnel interface (SVTI)
+
+< insert the instructions for creating the IPsec SVTI here. Below are some
+examples of parameters to set.>
+
+-  Adjust the maximum segment size (MSS) value of TCP packets going through a
+router as discussed in
+[MTU Considerations](https://cloud.google.com/vpn/docs/concepts/mtu-considerations)
+for Cloud VPN.
+
+```
+<insert configuration code snippet here>
+```
+
+### Configuring the dynamic routing protocol
+
+Follow the procedure in this section to configure dynamic routing for traffic
+through the VPN tunnel or tunnels using the BGP routing protocol.
+
+< insert the instructions for configuring dynamic routing here. Below are some
+examples of parameters to set.>
+
+BGP timers are adjusted to provide more rapid detection of outages.
```

Comment:
Is this something the customer should do? If so, I'd reword this to < insert instructions for setting BGP timers x, y, and z settings to x, y, and z values>. In other words, you need to be specific about what timers and what values to set on the peer side, unless this is determined entirely by the customer.  Plus add inserting a config code snippet for this, as well.